### PR TITLE
docs(misc): add version-support-status page

### LINKED ISSUE #16058: Supported Status for Versions
From: https://github.com/angular/angularjs.org/issues/220

Greetings -

It would be great if angularjs.org could maintain a simple page that indicates AngularJS versions and their EOL/Security Patch/LTS status.

For example, PHP has a similar page here: http://php.net/supported-versions.php

It clearly outlines the bug fix / security patch strategy for each supported version of PHP.

For angular, there are blog entries that elude to support (security fixes for 1.2.x in the 1.6.0 announcement post, for example). But even if there was just a list, not necessarily with timelines, of the most recent version of each significant release of Angular, and the current status of support for that release, it would make it simple for a developer to choose a supported version...

Or in our case, justify staying on a 1.2.x version because security patches are still maintained.

Thanks!

C

### LINKED ISSUE #16458: Update repo README and angularjs.org landing page with 1.7/LTS information
**I'm submitting a ...**
- [x] other

Hi team,
Can you consider adding a paragraph like this below one, taken from [Medium blog post](https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c):

> AngularJS is planning one more significant release, version 1.7, and on July 1, 2018 it will enter a 3 year Long Term Support period.

The information in this blog post has some significant value and repercussion and I think it should be somehow more prominent, better visible, allowing all interested parties to make better decision in future.

Thanks!

### PR DESCRIPTION
**Pull request by petebacondarwin**
Closes #16058
Closes #16458

### CODE REVIEW

**Comment by petebacondarwin** [comment]
screen shot 2018-02-19 at 17 13 46
screen shot 2018-02-19 at 17 13 31

**Comment by petebacondarwin** [comment]
This is the detail page.

screen shot 2018-02-19 at 17 13 46

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ It also helps with server-side communication, taming async callbacks with promis
 and it makes client-side navigation and deep linking with hashbang urls or HTML5 pushState a
 piece of cake. Best of all? It makes development fun!
 
+##### AngularJS will be moving to Long Term Support (LTS) mode on July 1st: [Find out more](misc/version-support-status)
+
+##### Looking for the new Angular? Go here: https://github.com/angular/angular
+
+--------------------
```
Comment (Narretz) [inline README.md]: Can you put another rule above AngularJS will be moving ... ? That'll frame it nicely

File: docs/content/misc/version-support-status.ngdoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This page describes the support status of the significant versions of AngularJS.
+
+<div class="alert alert-info">
+  AngularJS is planning one more significant release, version 1.7, and on July 1, 2018 it will enter a 3 year Long Term Support period.
+</div>
+
+### Until July 1st 2018
+
+Any version branch not shown (e.g. 1.5.x) is no longer being developed.
```
Comment (Narretz) [inline docs/content/misc/version-support-status.ngdoc]: ultra explicit: "not shown in this table"

File: docs/content/misc/version-support-status.ngdoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  <tr class="security"><td><span>1.2.x</span></td><td>Long Term Support</td><td>Last version to provide IE 8 support</td></tr>
+  <tr class="stable"><td><span>1.7.x</span></td><td>Long Term Support</td><td>1.7.0 will be the last release of AngularJS to contain breaking changes</td></tr>
+</tbody>
+</table>
+
+### Long Term Support
+
+On July 1st 2018, we will enter a Long Term Support period for AngularJS.
+
+At this time we will focus exclusively on providing fixes to bugs that satisfy at least one of the following criteria:
```
Comment (Narretz) [inline docs/content/misc/version-support-status.ngdoc]: Technically, also anything that is "easy" to add, e.g. small features, bug fixes, docs, iirc? Maybe just strip "exclusively", and then "focus" still leaves a lot of room for interpretation :)

File: docs/content/misc/version-support-status.ngdoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Any version branch not shown (e.g. 1.5.x) is no longer being developed.
+
+<table class="dev-status table table-bordered">
+<thead>
+  <tr><th>Version</th><th>Status</th><th>Comments</th></tr>
+</thead>
+<tbody>
+  <tr class="security"><td><span>1.2.x</span></td><td>Security patches only</td><td>Last version to provide IE 8 support</td></tr>
+  <tr class="stable"><td><span>1.6.x</span></td><td>Patch Releases</td><td>Minor features, bug fixes, security patches - no breaking changes</td></tr>
+  <tr class="current"><td><span>1.7.x</span></td><td>Active Development</td><td>1.7.0 (not yet released) will be the last release of AngularJS to contain breaking changes</td></tr>
```
Comment (Narretz) [inline docs/content/misc/version-support-status.ngdoc]: To "to contain breaking changes" maybe add "and significant features"?

File: docs/content/misc/version-support-status.ngdoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Any version branch not shown (e.g. 1.6.x) is no longer being developed.
+
+<table class="dev-status table table-bordered">
+<thead>
+  <tr><th>Version</th><th>Status</th><th>Comments</th></tr>
+</thead>
+<tbody>
+  <tr class="security"><td><span>1.2.x</span></td><td>Long Term Support</td><td>Last version to provide IE 8 support</td></tr>
+  <tr class="stable"><td><span>1.7.x</span></td><td>Long Term Support</td><td>1.7.0 will be the last release of AngularJS to contain breaking changes</td></tr>
```
Comment (Narretz) [inline docs/content/misc/version-support-status.ngdoc]: Is this copy and paste? I think the description should stress that 1.7.x will only receive maintenance. Or just refer to the section below.

**Review by Narretz — CHANGES_REQUESTED** [review]
Looking good, just some smaller change requests.
There is a section about the versioning in the FAQ. https://docs.angularjs.org/misc/faq#what-is-the-angularjs-versioning-strategy- 
Can you add a link from there to the new page?

File: docs/content/misc/version-support-status.ngdoc (hunk 4 above)
Comment (petebacondarwin) [reply docs/content/misc/version-support-status.ngdoc]: I am not sure if that adds much - "significant features" is pretty vague, and the statement as it stands does not preclude such things.

File: docs/content/misc/version-support-status.ngdoc (hunk 3 above)
Comment (petebacondarwin) [reply docs/content/misc/version-support-status.ngdoc]: I don't think so. Once we hit LTS we really do not intend to add anything other than stuff in this list.

File: docs/content/misc/version-support-status.ngdoc (hunk 5 above)
Comment (petebacondarwin) [reply docs/content/misc/version-support-status.ngdoc]: copy + paste :-)

**Comment by petebacondarwin** [comment]
@Narretz PTAL

**Review by Narretz — APPROVED** [review]

**Pull request merged**